symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -652,6 +655,33 @@ public FollowingListResponse listUsersFollowing(@Nonnull Long userId, @Nonnull C
     return new PaginatedService<>(api, pagination.getChunkSize(), pagination.getTotalSize()).stream();
   }
 
+  /**
+   * Return a {@link V2UserDetail} with created user details.
```

Comment:
I would say the method creates a new user (i.e returning something is already said by @return)